package.json

```diff
@@ -13,52 +13,58 @@
   },
   "scripts": {
     "lint": "eslint ui/**/*.vue ui/**/*.js --color --global requirejs --global require",
-    "watch": "webpack --progress --colors --watch --mode development --config webpack.js",
-    "build": "webpack -p --config webpack.js",
+    "build": "rollup -c",
+    "watch": "rollup -c -w",
     "test": "echo 'Not implemented'",
     "hello": "node node_modules/swagger-vue-generator/bin/generate-api.js --package-version v0 --source pkg/proto/v0/hello.swagger.json --moduleName hello --destination ui/client/hello/index.js"
   },
   "devDependencies": {
-    "@babel/cli": "^7.5.0",
-    "@babel/core": "^7.5.4",
-    "@babel/plugin-proposal-class-properties": "^7.5.0",
-    "@babel/plugin-proposal-export-default-from": "^7.2.0",
-    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
-    "@babel/plugin-transform-classes": "^7.4.4",
-    "@babel/plugin-transform-runtime": "^7.5.0",
-    "@babel/preset-env": "^7.4.5",
-    "@babel/register": "^7.4.4",
-    "@babel/runtime": "^7.5.4",
+    "@babel/core": "^7.7.7",
+    "@babel/preset-env": "^7.7.7",
+    "@rollup/plugin-commonjs": "^11.0.1",
+    "@rollup/plugin-json": "^4.0.1",
+    "@rollup/plugin-replace": "^2.3.0",
     "axios": "^0.19.0",
-    "babel-eslint": "^10.0.2",
-    "babel-loader": "^8.0.6",
     "core-js": "3",
-    "css-loader": "^3.0.0",
+    "cross-env": "^6.0.3",
     "debounce": "^1.2.0",
     "easygettext": "^2.7.0",
-    "eslint": "5.16.0",
-    "eslint-config-standard": "^12.0.0",
-    "eslint-loader": "^2.1.2",
+    "eslint": "6.8.0",
+    "eslint-config-standard": "^14.1.0",
     "eslint-plugin-import": "^2.17.3",
-    "eslint-plugin-node": "9.1.0",
+    "eslint-plugin-node": "11.0.0",
     "eslint-plugin-promise": "^4.1.1",
     "eslint-plugin-standard": "^4.0.0",
-    "eslint-plugin-vue": "^5.2.2",
+    "eslint-plugin-vue": "^6.1.2",
     "husky": ">=2",
     "lint-staged": ">=9",
     "qs": "^6.9.1",
     "rimraf": "^3.0.0",
+    "rollup": "^1.28.0",
+    "rollup-plugin-babel": "^4.3.3",
+    "rollup-plugin-eslint": "^7.0.0",
+    "rollup-plugin-filesize": "^6.2.1",
+    "rollup-plugin-node-builtins": "^2.1.2",
+    "rollup-plugin-node-globals": "^1.4.0",
+    "rollup-plugin-node-resolve": "^5.2.0",
+    "rollup-plugin-terser": "^5.1.3",
+    "rollup-plugin-vue": "^5.1.4",
     "swagger-vue-generator": "^1.0.6",
-    "vue-loader": "^15.7.0",
-    "vue-template-compiler": "2.6.10",
-    "vuex": "3.1.1",
-    "webpack": "4.35.3",
-    "webpack-cli": "3.3.4",
-    "webpack-merge": "4.2.1"
+    "vue-template-compiler": "^2.6.11"
   },
   "browserslist": [
-    "last 1 version",
     "> 1%",
-    "IE 11"
-  ]
+    "not dead"
+  ],
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```

Comment:
I dislike these commit hooks pretty much, IMHO it's up to the developer and CI to handle that. Especially the Go developers don't want to run the installed hooks on every Go change.

Reply:
Removed :wink: